solutions/bronze/usaco-1084.mdx

```diff
@@ -163,7 +163,7 @@ import java.util.*;
 public class OddPhotos {
 	public static void main(String[] args) throws IOException {
 		BufferedReader in =
-			new BufferedReader(new InputStreamReader(System.in));
+		    new BufferedReader(new InputStreamReader(System.in));
 		int odd = 0;
```

Comment:
WHY IS IT USING SPACES HERE WHAT THE HECK

Reply:
tabs are used for indentation only